### PR TITLE
FixerFactoryTest - add missing test

### PR DESCRIPTION
**Pull request by SpacePossum**
test fails locally on master only for me, not sure why it passes on travis currently

### CODE REVIEW

**Comment by keradus** [comment]
That's weird. Let us try to fix the cause first, symptom later. If we have instability in tests, I want to track it down.
`master` branch is passing on my local machine as well, so it's not only CI.

Upgrade master & deps and try again. If issue still occurs, please provide version of your OS, PHP and PHPUnit.

**Comment by keradus** [comment]
OK, files from `/priorities/` are loaded in random order.

reason: `testPriorityIntegrationTestFilesAreListedPriorityCases`
it's one test with loop inside instead of test with data provider.

when it hit `markTestIncomplete`, it stops testing rest of the cases

**Comment by keradus** [comment]
2.2 shall be the target IMO

**Comment by SpacePossum** [comment]
I don't think this issue happens on 2.2

**Comment by keradus** [comment]
awesome :D 
i wanted to be sure that we won't hit this issue again, as we do fixes on 2.2 but don't verify they are fully covered

File: tests/AutoReview/FixerFactoryTest.php [hunk 1]
```diff
@@ -283,40 +283,76 @@ function (array $case) use ($self) {
...
+                }
+
+                $priorityCases[$fixerName][$priorityCase[1]->getName()] = true;
             }
 
-            $priorityCases[$fixerName][$priorityCase[1]->getName()] = true;
+            ksort($priorityCases);
+        }
+
+        if ('braces,indentation_type,no_break_comment.test' === $fileName) {
```
[the remark this answers is not in the document]
Comment (keradus) [reply tests/AutoReview/FixerFactoryTest.php]: and on master we already have it:
```
tests/AutoReview/FixerFactoryTest.php-            if (in_array($fileName, [
tests/AutoReview/FixerFactoryTest.php:                'braces,indentation_type,no_break_comment.test',
tests/AutoReview/FixerFactoryTest.php-            ], true)) {
tests/AutoReview/FixerFactoryTest.php-                $this->markTestIncomplete(sprintf('Case "%s" has unexpected name, please help fixing it.', $fileName));
```

no need for adding it here, on 2.2 it's dead code.

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request closed without merging**